src/site/antora/modules/ROOT/pages/manual/index.adoc

```diff
@@ -16,122 +16,52 @@
-xref:manual/filters.adoc[Filters] and xref:manual/lookups.adoc[Lookups] to manipulate
-them.
-9.  Log4j 1.x supports Filters on Appenders. Logback added TurboFilters
-to allow filtering of events before they are processed by a Logger.
-Log4j 2 supports Filters that can be configured to process events before
-they are handled by a Logger, as they are processed by a Logger or on an
-Appender.
-10. Many Logback Appenders do not accept a Layout and will only send
-data in a fixed format. Most Log4j 2 Appenders accept a Layout, allowing
-the data to be transported in any format desired.
-11. Layouts in Log4j 1.x and Logback return a String. This resulted in
-the problems discussed at
-http://logback.qos.ch/manual/encoders.html[Logback Encoders]. Log4j 2
-takes the simpler approach that xref:manual/layouts.adoc[Layouts] always return
-a byte array. This has the advantage that it means they can be used in
-virtually any Appender, not just the ones that write to an OutputStream.
-12. The xref:manual/appenders.adoc#SyslogAppender[Syslog Appender] supports
-both TCP and UDP as well as support for the BSD syslog and the
-http://tools.ietf.org/html/rfc5424[RFC 5424] formats.
-13. Log4j 2 takes advantage of Java 5 concurrency support and performs
-locking at the lowest level possible. Log4j 1.x has known deadlock
-issues. Many of these are fixed in Logback but many Logback classes
-still require synchronization at a fairly high level.
-14. It is an Apache Software Foundation project following the community
-and support model used by all ASF projects. If you want to contribute or
-gain the right to commit changes just follow the path outlined at
-http://jakarta.apache.org/site/contributing.html[Contributing].
+== Welcome to Apache Log4j!
+
+Apache Log4j is a versatile, industrial-grade Java logging framework composed of an API, its implementation,  and components to assist the deployment for various use cases.
+Log4j is https://security.googleblog.com/2021/12/apache-log4j-vulnerability.html[used by 8% of the Maven ecosystem] and listed as one of https://docs.google.com/spreadsheets/d/1ONZ4qeMq8xmeCHX03lIgIYE4MEXVfVL6oj05lbuXTDM/edit#gid=1024997528[the top 100 critical open source software projects].
+The project is actively maintained by a {logging-services-url}/team-list.html[team] of several volunteers and {logging-services-url}/support[support]ed by a big community.
+
+Logging is an essential part of the software development process. 
+It provides a way to track the flow of execution in a program, allowing developers 
+to understand the application's behavior without needing a debugger. 
+This is particularly useful when tracking bugs or understanding why a particular code runs slowly.
+
+The original concept for Log4j was conceived in early 1996 when the 
+E.U. SEMPER project decided to develop its own tracing API. 
+In 2003, the project was donated to the Apache Software Foundation, which became Apache Log4j. 
```

Comment:
s/which became Apache Log4j/becoming _Apache Log4j_/

Reply:
Thanks @Chealer . if you like, please create a PR and you can see your contribution in this file :-)

Reply:
@Chealer,

You can follow our documentation rewrite on our [staging website](https://logging.staged.apache.org/log4j/2.x/index.html).
If we there are typos, grammar or other problems, feel free to use the "Edit" button on the top right side of the page to modify the page in Github and submit a PR.

**Note**: not all the pages have been rewritten yet. At this moment only the #2538 and #2539 pages are new. If you edit other pages, your changes might be lost, when we rewrite them.

Reply:
I made a few remarks which require someone more involved (ideally the author) to intervene, but if I still notice simple issues after that's solved, thanks, I will consider.
Thank you for the link @ppkarwasz !

Reply:
No problem, I can work through the comments - thanks for adding them @Chealer 
Whenever you want to get more involved, please let me or @ppkarwasz know, we are happy about everyone who wants to get their hands dirty :)